web-ng/src/app/components/drawing-tools/drawing-tools.component.css

```diff
@@ -24,6 +24,6 @@
   position: absolute;
   top: 2em;
   left: 50%;
-  background-color: #fff;
+  background-color: white;
```

Comment:
I was referring to using the `mat-color()` directive, but since this is a temp UI we can leave it as-is for now.

Reply:
Cool, I'll leave it as-is